src/ci-line-chart.js

```diff
@@ -226,12 +228,17 @@ const vis = {
           x: {
             type: 'time',
             time: {
-              tooltipFormat: 'DD T'
+              tooltipFormat: 'DD',
+              unit: 'day',
+              round: 'day',
             },
             title: {
               display: true,
               text: optionsToFriendly[config.field_x]
-            }
+            },
+            grid: {
+              display: false
+            },
```

Comment:
I think we might want to make this configurable, but at least for opmon the default is to set this off

Reply:
Added a ticket (#18)